### ISSUE TITLE
$XDG_CONFIG_HOME support for registries.conf

### ISSUE THREAD

**Issue by p0da** [containers/podman#15475]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

Would you consider adding $XDG_CONFIG_HOME support to podman in order for it to locate registries.conf files located at a specified XDG_CONFIG_HOME. Files like storage.conf seem to also support this behavior while registries.conf does not for some reason.

**Comment by vrothberg** [comment]
Thanks for reaching out, @p0da.  I recall conversations about that a longer while ago but my memory is fading.

@mtrmac has a much better memory :)

**Comment by mtrmac** [comment]
I can’t see any earlier _design_ conversation about this, just a few offhand comments stating that that’s what it is.

The default location is already consistent with $XDG_CONFIG_HOME use, so switching should be seamless for default installs that don’t modify the variable (but it would, of course, break those use a different value of $XDG_CONFIG_HOME currently).

I really have no preference about this at all. Purely personally — if you care to do the work of changing the code and updating documentation throughout, why not.

**Comment by p0da** [comment]
Makes sense. I would say that from a consistency POV this is fairly important as it is feels very unintuitive to have some files work with a different $XDG_CONFIG_HOME while others don't.
